client/src/app/all-projects/all-projects.component.ts

```diff
@@ -69,62 +74,46 @@ export class AllProjectsComponent implements OnInit, OnDestroy {
   getProjects(params) {
     const urlParams = {
       ...params,
+      ...params?.filterState,
       sort: `${params.sort.column},${params.sort.direction}`
     };
-    if (params.filterState) {
-      return this.searchProjects(urlParams);
-    }
-    return this.getDefaultProjects(urlParams);
-  }
 
-  searchProjects(params) {
-    const query = [];
-    const fields = [
-      'content.project_core.project_description',
-      'content.project_core.project_title',
-      'content.project_core.project_short_name'
-    ];
-
-    for (const field of fields) {
-      const criteria = {
-        'field': field,
-        'operator': 'REGEX',
-        'value': params.filterState.replace(/\s+/g, '\\s+')
-      };
-      query.push(criteria);
-    }
-
-    delete params.filterState;
-
-    params['operator'] = 'or';
-    return this.queryProjects(query, params);
+    return this.getFilteredProjects(urlParams);
   }
 
-  getDefaultProjects(params) {
-    const criteria = {
-      field: 'isUpdate',
-      operator: 'IS',
-      value: false
-    } as Criteria;
-    params['operator'] = 'and';
-    return this.queryProjects([criteria], params);
+  private getFilteredProjects(params) {
+    return this.ingestService.getFilteredProjects(params).pipe(map(
+      data => {
+        const pagedData: PagedData<MetadataDocument> = {data: [], page: undefined};
+        pagedData.data = data._embedded ? data._embedded.projects : [];
```

Comment:
You could use the elvis operator here.
```suggestion
        pagedData.data = data._embedded?.projects || [];
```